rpm/prebuild.sh

```diff
@@ -0,0 +1,37 @@
+set_python3() {
+    python3_default=/usr/bin/python3
+    for v in `seq 9 -1 4`
```

Comment:
Python 3.10 is planned at October, 2011 and it'll likely picked up by Fedora quickly.

Reply:
You probably mean 2021. I see release date is set to March, 2021 in [1].
Anyway I got what you wanted to say :)

1. https://docs.python.org/3.10/whatsnew/3.10.html

Reply:
Since the prebuild script is gone, it does not look relevant anymore.